PetaPoco.Tests.Integration/Databases/UpdateTests.cs

```diff
@@ -81,6 +81,8 @@ protected static void UpdateProperties(OrderLine orderLine)
             orderLine.Status = OrderLineStatus.Allocated;
         }
 
+        protected object CreateNote() => DB.Insert(_note);
+        protected Task<object> CreateNoteAsync() => DB.InsertAsync(_note);
```

Comment:
This looked like a factory creation method initially; something more like `CreateAndInsertNote()` would be more communicative of the intent, but frankly I don't see any justification in separating it out of the test methods: it obscures what's going on in the test, and fails to serve any purpose besides forwarding the call.

Reply:
Only did it cause I didn't want to make _note protected. Similar to what you did to GetDatabase(), unless you're ok with protected. Not sure what will happend with the test data looking at the TODO statement

Reply:
OK, makes more sense in light of that. Changing visibility to protected would be preferable at this point (and I'm doing that on other derived test classes). Overriding test methods should be tested against the same fixture as the base test methods anyways, in most cases.

Reply:
Dope! I should remove the underscore, shouldn't I?

Reply:
That is correct. Cheers.